lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -499,38 +501,42 @@ export abstract class CacheManager implements ICacheManager {
     /**
      * Removes all accounts and related tokens from cache.
      */
-    removeAllAccounts(): boolean {
+    async removeAllAccounts(): Promise<boolean> {
```

Comment:
Same question re: breaking change

Reply:
`removeAllAccounts is called by the `BrowserCacheManager` so I think this is a breaking change. How big of a deal would it be to make a major version bump for common? I don't believe there's anyway to correctly implement logout without making this `async`.

Reply:
Not a big deal if this it needed to do this properly.